clients/include/unit.hpp

```diff
@@ -409,4 +409,13 @@ inline void trsm_err_res_check(T max_error, rocblas_int M, T forward_tolerance,
 #endif
 }
 
+template <typename T>
+double get_epsilon()
```

Comment:
I noticed some helpers were being created to help identity the underlying type in complex cases. I was thinking of switching over to using these helpers instead of creating a new helper for the epsilon. You can see the addition here:
https://github.com/ROCmSoftwarePlatform/rocBLAS/blob/8d8d415db5c04c62e835dca68948b3d0aa2d6522/library/src/include/utility.h#L367

Reply:
Will switch to use specializations once Daine's PR is merged